acceptance/setup/aio/010_Install.rb

```diff
@@ -23,11 +30,11 @@
       on(master, "rpm -Uvh https://yum.puppetlabs.com/puppet5-release-el-#{version}.noarch.rpm")
     end
   else
-    if ENV['SERVER_VERSION']
-      install_from_build_data_url('puppetserver', "#{dev_builds_url}/puppetserver/#{ENV['SERVER_VERSION']}/artifacts/#{ENV['SERVER_VERSION']}.yaml", master)
-    else
-      install_puppetlabs_dev_repo(master, 'puppetserver', 'latest', nil, :dev_builds_url => 'http://nightlies.puppetlabs.com')
+    if dev_builds_url == 'http://nightlies.puppetlabs.com'
```

Comment:
Will this work in existing CI? I'm not sure what we set for AWS or Jenkins test environments.